### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by kgajriaAgarwal**
1. Created the Homepage 
image
2. Created the quizes category page
image
3. Created rules page
image
Kindly review it and let me know what do you think about it!

### CODE REVIEW

File: Categories/Categories.html [hunk 1]
```diff
@@ -40,7 +40,7 @@ <h2 class="header-title">
                     <p class="text-sm card-des">Covid Essentials</h4>
```
Comment (Ankur9669) [inline Categories/Categories.html:40]: You html and css file name are starting from capital letters.
It is recommended to use only small case characters for html and css files

**Review by manish-GH — APPROVED** [review]
Ohh it's just small changes in this PR. Yeah, everything looks good. 

The UI looks great on the pictures attached. Maybe you could consider adding some spacing between the icon and avatar on the navbar

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
image
The margin is not consistent.

image
There is a little shift when we hover over the button inside card, is it intensional??

I have reviewed but most of the code is not visible there are only 3 files changed and all code is not visible could you check it

File: Categories/Categories.html (hunk 1 above)
Comment (kgajriaAgarwal) [reply Categories/Categories.html:40]: I suppose this will not matter as we are going to convert these  pages into reusable components.

**Pull request merged**